types/UIShell/HeaderGlobalAction.svelte.d.ts

```diff
@@ -18,13 +18,11 @@ export interface HeaderGlobalActionProps extends ButtonProps {
    * Obtain a reference to the HTML button element
    * @default null
    */
-  ref?: null | HTMLButtonElement;
-
-  [key: `data-${string}`]: any;
+  ref?: undefined;
```

Comment:
I am not sure why it was updated this way by `yarn build:docs`. Should I edit this file manually?

Reply:
I applied this suggestion, but the .d.ts file did not update properly until I specified the type in JSDoc explicitly